DESCRIPTION

```diff
@@ -59,7 +59,9 @@ Imports:
     secr,
     sp,
     stringi,
-    stringr
+    stringr,
+    covr,
+    testthat
```

Comment:
Thinking about this more, `covr` and `testthat` should probably be in Suggests rather than imports because, strictly speaking, they aren't required for core `SpaDES` functionality but instead are only used for module development by the user (and for package development).